draft-ietf-masque-h3-datagram.md

```diff
@@ -262,15 +284,15 @@ Capsule Value:
 : The payload of this capsule. Its semantics are determined by the value of the
 Capsule Type field.
 
-Because new protocols or extensions may involve defining new capsule types,
+An intermediary can identify the use of the capsule protocol either through the
+presence of the Capsule-Protocol header field ({{hdr}}) or by understanding the
+chosen HTTP Upgrade token.
+
+Because new protocols or extensions might define new capsule types,
 intermediaries that wish to allow for future extensibility SHOULD forward
-capsules unmodified. One exception to this rule is the DATAGRAM capsule; see
-{{datagram-capsule}}. An intermediary can identify the use of the capsule
-protocol either through the presence of the Capsule-Protocol header field
-({{hdr}}) or by understanding the chosen HTTP Upgrade token. An intermediary
-that identifies the use of the capsule protocol MAY convert between DATAGRAM
-capsules and QUIC DATAGRAM frames when forwarding. Definitions of new Capsule
-Types MAY specify optional custom intermediary processing.
+capsules without modification, unless the definition of the Capsule Type in use
+specifies additional intermediary processing. One such Capsule Type is the
+DATAGRAM capsule; see {{datagram-capsule}}.
 
 Endpoints which receive a Capsule with an unknown Capsule Type MUST silently
```

Comment:
Use of the term 'endpoints' in the spec is a tiny bit sloppy - sometimes, it's referring to the transport-layer endpoints, others the HTTP endpoints (i.e., user agent and origin server). Here, I think it's the latter, but many transport-oriented readers may assume the former. Either paranthetically reminding readers or choosing different terms might help.

Reply:
Fixed by removing many instances of that word